src/components/periodic-table/PeriodicTable.tsx

```diff
@@ -59,7 +59,7 @@ class PeriodicTable extends React.PureComponent<
 
     if (!render) {
       return (
-        <div className="periodic-table--loading">
+        <div className="periodic-table--loading" aria-label="loading-spinner">
```

Comment:
Aria label is what the screen readers will hear, it is better to just put "loading" in here, "loading-spinner" can get confusing. :tada: 

So when the data is getting loaded, it will say "loading" and not "loading-spinner", which sounds like it is loading some sort of spinner, but what is getting loaded is just the data.

Reply:
Would the `aria-label={`element-${element}`}` also read a bit funny too then?

Reply:
@mpeyper Likely yes!!

Thanks, I missed that completely :rofl: :pray: 

@tsuki42 Could change that as well? :blue_heart:

Should be on line 164.

Reply:
I was thinking in terms of chemical elements. How does changing it to  `atomic-number` seem?

Reply:
@tsuki42 I read here https://www.aoda.ca/accessible-writing-style/:

> Additionally, writers must be aware that spelling errors will be distracting, especially to people who use screen readers. Peer editing, self-review, and the use of spelling and grammar-checking software, such as spell-check, is vital.

My advice would be that we skip it for now?

We can always improve it later :+1:

I just feel like this would be a use-case where you need to dig a bit deeper into accessibility and take your time :man_shrugging:

Reply:
I needed something to query elements by. Instead of using data-testid, I chose this.

Reply:
Try Testing Playground, you can get the URL via `screen.logTestingPlayground`.

If test id is really needed, then we can just add one I guess :+1: